ops/pebble.py

```diff
@@ -1103,6 +1106,11 @@ def __init__(self, name: str, raw: Optional[CheckDict] = None):
         except ValueError:
             level = dct.get('level', '')
         self.level = level
+        try:
+            startup: Union[CheckStartup, str] = CheckStartup(dct.get('startup', ''))
```

Comment:
I realise we decided not to remove the union for existing fields, but for new fields shall we simplify and just use `startup: CheckStartup` (where one of the options is `''`)? It would avoid awkward types and awkward `isinstance` checks like we see below -- at least for the new field.

Reply:
Sure - I wondered about that but thought maybe it would be better to just be consistent. But happy to go with the simpler case.